### ISSUE TITLE
AFK push improvement

### ISSUE THREAD

**Issue by ArtjomsBrovkins** [Fortigate/AutoAFK2#21]
From what I see current implementation for AFK stage push only takes into account latest season with talents/regular afk double-stages.

I think it would be better to refactor the code a bit to allow single stage push as well for the beginning of the latest season or previous season. I would guess that distinction between single-team/double-team can be made based on visibility of team selector on a battle screen. It will also allow to implement more stages in the future, as I would guess there will be 3+ fights per stage in the future.

Not sure how it would work in pre-season, as from what I remember, there could be up to 3 stages to select from.

**Comment by Fortigate** [comment]
Found a fairly painless one-line solution for this, when checking if the first stage is complete we check for the formation swap button as ell, if that isn't visible we set the first_stage_complete flag and so it checks for the standard victory screen. The text will say "Second stage complete!" which is a bit misleading but it should detect everything automagically.

Will run it on my internal test build on my accounts for a few days and it'll be in the next release.
